scripts/tools/EXT_Exp_MetricTracer.rst

```diff
@@ -0,0 +1,167 @@
+<%
+import re
+from templates import helper as th
+%><%
+    OneApi=tags['$OneApi']
+    x=tags['$x']
+    X=x.upper()
+    t=tags['$t']
+    T=t.upper()
+%>
+:orphan:
+
+.. _ZET_experimental_metric_tracer:
+
+==========================================
+Metric Tracer Experimental Extension
+==========================================
+
+API
+----
+* Enumerations
```

Comment:
we also need an enum for the extension version here

Reply:
added.